lighthouse-core/scripts/roll-to-devtools.sh

```diff
@@ -53,6 +50,6 @@ rsync -avh "$lh_locales_dir" "$fe_locales_dir" --exclude="*.ctc.json" --delete
 echo -e "$check Locale JSON files copied."
 
 echo ""
-echo "Done. To rebase the test expectations, run: "
-echo "    yarn --cwd ~/chromium/src/third_party/devtools-frontend/src test 'http/tests/devtools/lighthouse/*.js' --layout-tests-dir test/webtests --reset-results"
+echo "Done. To rebase the test expectations, run (note: you must use the integrated checkout): "
```

Comment:
I hope this is temporary, because using the integrated checkout is so much overhead.